python/py-construct/Portfile

```diff
@@ -32,7 +31,7 @@ homepage            https://construct.readthedocs.org
 
 if {${name} ne ${subport}} {
 
-    if { ${python.version} == 27 || ${python.version} == 35} {
+    if { ${python.version} == 27} {
```

Comment:
Should this be removed?

Reply:
no, the py27 subport will need to stay at least for a while since it has a dependent:
```
% port echo depends:py27-construct
gr37-satellites
```

Reply:
Gotcha. Somehow I hadn't made the connection that you're removing Py27 ports with no dependencies ... makes sense now!